PocketBeagle/.work-in-progress/Grove/KeyBoard_Player.py

```diff
@@ -64,14 +67,10 @@ def __init__(self):
                     'if=/lib/firmware/BB-I2C2-MPR121.dtbo'])
                 time.sleep(0.1)
             if not os.path.exists('/sys/devices/platform/ocp/4819c000.i2c/i2c-2/2-005b/mpr121_data'):
-                subprocess.call(['sudo', 'modprobe', 'hd44780'])
-                time.sleep(0.1)
-            try:
-                self.f = open('/sys/devices/platform/ocp/4819c000.i2c/i2c-2/2-005b/mpr121_data', 'r')
-            except IOError as err:
-                subprocess.call(['sudo', 'chmod', '777',
-                    '/sys/devices/platform/ocp/4819c000.i2c/i2c-2/2-005b/mpr121_data'])
-                self.f = open('/sys/devices/platform/ocp/4819c000.i2c/i2c-2/2-005b/mpr121_data', 'r')
+                mod_path = '·'+GetCmdReturn('uname -r')+'/extra/seeed/mpr121.ko'       
```

Comment:
Since we have this pattern in so many places, maybe we should make a class/module for it?